### DISCUSSION TITLE
Unresolved Alias

### DISCUSSION THREAD

**Discussion by johnwargo** [eemeli/yaml#454] [General]
I'm parsing some eleventy markdown documents to read the frontmatter using YAML and when my code gets to one of the documents, I get the following error: `Unresolved alias (the anchor must be set before the alias)`. What does this mean? I've looked at my document content and I can't identify the issue.

**Comment by eemeli** [comment]
Your document contains an alias like `*foo` that's attempting to refer to a node with an anchor like `&foo`, which must be before the alias in the document.
